plugins/git/current_git.mli

```diff
@@ -40,7 +40,8 @@ module Commit : sig
 end
 
 val clone : schedule:Current_cache.Schedule.t -> ?gref:string -> string -> Commit.t Current.t
-(** [clone ~schedule ~gref uri] evaluates to the head commit of [uri]'s [gref] branch (default: "master"). *)
+(** [clone ~schedule ~gref uri] evaluates to the head commit of [uri]'s [gref] branch (default: make 
+    use of `git ls-remote` to get the remote's default branch). *)
```

Comment:
At best the default git branch is mix of `master/main/trunk` the only reliable way to know which one to choose is to already know it or to query the Git hosting platform to tell you (GitHub/GitLab both have API calls to find this). Rather than making an educated guess here it would be better to keep the default as is, forcing an API change when you *must* supply a default branch seems unnecessary.